todoCLI/todoCLI.js

```diff
@@ -0,0 +1,153 @@
+const rl = require(`readline`);
+const fs = require(`fs`);
+
+const menu= `(V) View ✏︎ (n) New ✏︎ (cX) Complete  ✏︎  (dX) Delete ✏︎ (q) Quit\n`;
+
+console.log(`\n\nWelcome to Todo CLI!\n----------------------`);
+const interface = rl.createInterface ( {
+	input: process.stdin,
+	output: process.stdout
+});
+
+waitingForUser();
+
+function waitingForUser () {
+  interface.question(menu, (command) => {
+    let number = parseFloat(command.replace(/[cd]/ , ''))
+    command = command.replace(number, '')
+    switch (command) {
+      case `q`:
+        process.exit();
+        break;
+      case `V`:
+        viewToDoList();
+        break;
+      case `n`:
+        addTask();
+        break;
+      case `c`:
+        completeTask(number);
+        break; 
+      case `d`:
+        deleteTask(number);
+        // console.log(`You typed dX`);
+        break;
+      default:
+        console.log(`Invalid entry\n`);
+        waitingForUser();
+        break;
+      }
+  });
```

Comment:
I don't think you need call to `waitingForUser()` again and again... you're creating a recursive set of calls that go deeper and deeper every time. I think you just need a loop whose exit condition is if the command is `q`

Reply:
You're absolutely right, my bad. I didn't realize that `interface.question` was asynchronous.